doc/release/technotes/README.libraries.rst

```diff
@@ -3,9 +3,12 @@
 Chapel Libraries
 ================
 
-[This readme discusses the implementation of libraries, which are still under
-development.  The idea is to allow components to be developed in Chapel and
-linked with other programs and languages.]
+.. note::
+
+   This page discusses the implementation of libraries, which are still
+   under development.  The idea is to allow components to be developed in
+   Chapel and linked with other programs and languages.  It is very fragile
+   and very difficult to work with, so user beware.
```

Comment:
Could you say "It is currently fragile and difficult to work with" e.g. instead? I'd like it to be a bit more optimistic somehow..